site/en/blog/fedcm-auto-reauthn-origin-trial/index.md

```diff
@@ -0,0 +1,177 @@
+  Introducing a few updates for FedCM including a new origin trial "auto reauthentication", which reauthenticates users automatically (when RPs opt-in), when they come back after their initial authentication using FedCM.
+subhead: >
+  Introducing a few updates for FedCM including a new origin trial "auto reauthentication".
+date: 2023-03-09
+hero: 'image/YLflGBAPWecgtKJLqCJHSzHqe2J2/9C9Sneo3nkZlqRcsbWff.jpg'
+alt: 'an automated door'
+authors:
+  - agektmr
+origin_trial:
+  url: /origintrials/#/view_trial/2426314299245854721
+tags:
+  - privacy
+  - identity
+  - origin-trials
+---
+
+[Federated Credential Management API (FedCM)](/docs/privacy-sandbox/fedcm/) is a
+web API for privacy-preserving identity federation. With identity federation, an
+RP (relying party) relies on an IdP (identity provider) to provide the user an
+account without requiring a new username and password.
+
+FedCM is a purpose-built API that allows the browser to understand the context
+in which the RP and IdP exchange information, inform the user as to the
+information and privilege levels being shared and prevent unintended abuse.
+
+## Updates
+
+We have a few updates to the Chrome's FedCM implementation recently:
+
+* For the [ID assertion
+  endpoint](/docs/privacy-sandbox/fedcm/#id-assertion-endpoint), IdPs need to
+  check the `Origin` header (instead of the `Referer` header) to see if the
+  value matches the origin of the client ID.
+* Added a new [Chrome flag](/docs/web-platform/chrome-flags/)
+  `chrome://flags/#fedcm-without-third-party-cookies`. With this flag, [you can
+  test FedCM functionality in Chrome by blocking third-party
+  cookies](/docs/privacy-sandbox/fedcm/#block-third-party-cookies).
+
+Check all the past updates to the API and our [FedCM
+documentation](/docs/privacy-sandbox/fedcm/) at [Federated Credential Management
+API updates](/docs/privacy-sandbox/fedcm-updates/).
+
+Additionally, we are starting FedCM's new feature as an origin trial — "auto
+reauthentication" — starting in Chrome 112.
+
+## Auto-reauthentication
+
+Currently, after [a user has created a federated account on an RP with an IdP
+via the FedCM](/docs/privacy-sandbox/fedcm/#sign-in), the next time they revisit
+the website they need to go through the same steps with the same UI affordances.
+i.e. they will need to explicitly and manually re-confirm to re-authenticate and
+proceed with the sign-in flow. While the explicit UX makes sense before the user
+has created the federated account to prevent tracking (which is one of the main
+goals of FedCM), it is overly and unnecessarily cumbersome after the user has
+gone through it once: after the user grants permission to allow communication
+between the RP and the IdP,  there's no privacy or security benefit for
+enforcing another explicit user confirmation for something that they have
+already previously acknowledged. Therefore we are introducing a more streamlined
+UX that RPs can choose for their returning users.
```

Comment:
```suggestion
Currently, after a user has [created a federated account on an RP with an IdP
via the FedCM](/docs/privacy-sandbox/fedcm/#sign-in), the next time they visit
the website they need to go through the same steps in the user interface.
That is, they need to explicitly confirm and reauthenticate to
proceed with the sign-in flow. As one of the main
goals of FedCM is to prevent covert tracking, this user experience (UX) makes sense before the user
has created the federated account, but it becomes unnecessary and cumbersome after the user has
gone through it once. After the user grants permission to allow communication
between the RP and the IdP,  there's no privacy or security benefit for
enforcing another explicit user confirmation for something that they have
already previously acknowledged. That's why Chrome is introducing a more streamlined
UX that RPs can choose for their returning users.
```